test/e2e/framework/util.go

```diff
@@ -286,95 +286,99 @@ func FailfWithOffset(offset int, format string, args ...interface{}) {
 	ginkgowrapper.Fail(nowStamp()+": "+msg, 1+offset)
 }
 
-func Skipf(format string, args ...interface{}) {
+func skipInternalf(caller int, format string, args ...interface{}) {
```

Comment:
why do we need this version with `caller` as an argument? it looks like it's always called with 1.

Reply:
A helper that called a helper that calls skip would need to call it with 2.  We don't have one yet, but I was planning on adding one for cleaning up some of the ugly skip code in the apimachinery tests.  I could change this in the follow up, but anyone who hardcodes caller count is usually wrong.